addons/web/static/src/less/views.less

```diff
@@ -70,3 +70,11 @@
 .oe_view.oe_cannot_create .oe_view_nocontent_create {
     display: none;
 }
+
+// text color light green
```

Comment:
Furthermore, that way you have a strong rule for all fa-circle icons, as the text-success is originally a dark green one.